pkgs/development/interpreters/clojure/obb.nix

```diff
@@ -0,0 +1,72 @@
+{ lib
+, stdenv
+, fetchurl
+, babashka
+, cacert
+, clojure
+, git
+, jdk
+, obb
+, runCommand}:
+
+stdenv.mkDerivation rec {
+  pname = "obb";
+  version = "0.0.1";
+
+  src = fetchurl {
+    url = "https://github.com/babashka/${pname}/archive/refs/tags/v${version}.tar.gz";
+    sha256 = "sha256-ZVd3VCJ7vdQGQ7iY5v2b+gRX/Ni0/03hzqBElqpPvpI=";
+  };
+
+  buildInputs = [ babashka cacert git jdk ];
+
+  configurePhase = ''
+  mkdir -p .m2
+  substituteInPlace deps.edn --replace ':paths' ':mvn/local-repo "./.m2" :paths'
+  substituteInPlace bb.edn --replace ':paths' ':mvn/local-repo "./.m2" :paths'
+  echo deps.edn
+  '';
+
+  buildPhase = ''
+  export DEPS_CLJ_TOOLS_DIR=${clojure}
+  export DEPS_CLJ_TOOLS_VERSION=${clojure.version}
+  mkdir -p .gitlibs
+  mkdir -p .cpcache
+  export GITLIBS=.gitlibs
+  export CLJ_CACHE=.cpcache
+
+  bb build
+  '';
+
+  installPhase = ''
+  runHook preInstall
+
+  install -Dm755 "out/bin/obb" "$out/bin/obb"
+
+  runHook postInstall
+  '';
+
+  installCheckPhase = ''
+    $out/bin/obb --version | grep '${version}'
+    $out/bin/obb -e '(+ 1 2)' | grep '3'
+  '';
+
+  passthru.tests = {
+    simple = runCommand "${pname}-test" {} ''
+      ${obb} -e '(+ 1 2)' > $out
+      [ $(cat $out) = '3' ]
+      '';
+  };
+
+
+  meta = with lib; {
+    description = "Ad-hoc ClojureScript scripting of Mac applications via Apple's Open Scripting Architecture";
+    homepage = "https://github.com/babashka/obb";
+    license = licenses.epl10;
+    maintainers = with maintainers; [
+      thiagokokada
```

Comment:
I don't have a Mac so I am not interested in maintaining this package, sorry.
```suggestion
```

Reply:
Okay! Thanks for reviewing!